src/nway/transforms.py

```diff
@@ -111,6 +111,41 @@ def apply(self, src: np.ndarray, dst: np.ndarray
         return mov, fix
 
 
+def pad_to_match(arr1: np.ndarray, arr2: np.ndarray
+                 ) -> Tuple[np.ndarray, np.ndarray]:
+    """
+    given a mismatch in size, per-axis, pad the smaller array
+    to match the larger array size.
+
```

Comment:
Maybe add a note that the padding is appended and not inserted?

Reply:
done